src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -234,21 +234,6 @@ const useGroupInfo = (channelId = null) => {
       );
       setNewParticipant(updateParticipant);
       updateMemberName(result.members);
-      await channel.sendMessage(
-        {
-          text: `${profile.myProfile.username} removed ${
-            selectedUser.user.name || selectedUser?.user?.username
-          } from this group`,
-          isRemoveMember: true,
-          user_id: profile.myProfile.user_id,
-          silent: true
-        },
-        {skip_push: true}
-      );
-      await generateSystemChat(
-        `${profile.myProfile.username} removed you from this group`,
-        selectedUser.user_id || selectedUser.userId
-      );
       setNewParticipant(result.members);
       setParticipants(result.members, groupPatchDispatch);
     } catch (e) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `sendMessage` and `generateSystemChat` functions in this section seems to be part of a larger refactoring effort to handle system messages differently. However, it's important to ensure that the functionality provided by these removed lines is still covered elsewhere in your codebase. If not, you may lose the ability to notify users when they are removed from a group.

```diff
-      await channel.sendMessage(
-        {
-          text: `${profile.myProfile.username} removed ${
-            selectedUser.user.name || selectedUser?.user?.username
-          } from this group`,
-          isRemoveMember: true,
-          user_id: profile.myProfile.user_id,
-          silent: true
-        },
-        {skip_push: true}
-      );
-      await generateSystemChat(
-        `${profile.myProfile.username} removed you from this group`,
-        selectedUser.user_id || selectedUser.userId
-      );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->